bicep/main.bicep

```diff
@@ -70,17 +72,45 @@ resource existingAGWSubnet 'Microsoft.Network/virtualNetworks/subnets@2020-11-01
 }
 
 //------------------------------------------------------ Create custom vnet
+@minLength(9)
+@maxLength(18)
+@description('The address range for the AKS vnet')
```

Comment:
There's more than just AKS in the vnet. Can you change to;
"The address range for the custom vnet"

All other metadata looks great 👍